packages/tools/pie-icons-webc/playwright-react.config.ts

```diff
@@ -4,6 +4,8 @@ import path from 'path';
 export default defineConfig({
     /* The base directory, relative to the config file, for snapshot files created with toMatchSnapshot and toHaveScreenshot. */
     snapshotDir: './__snapshots__',
+    /* The template for the snapshot file path. */
+    snapshotPathTemplate: '{snapshotDir}/{testFilePath}/{arg}-{projectName}{ext}',
```

Comment:
Updating the naming of our screenshot names. Prior to this PR, the architecture was included, e.g.

MacOS - IconAlcoholXXL-component-chrome-darwin.png
Linux - IconAlcoholXXL-component-chrome-linux.png
Windows - IconAlcoholXXL-component-chrome-win32.png

By removing the platform name, we ensure these tests will pass on CI